mod_ci/models.py

```diff
@@ -15,6 +15,16 @@
 from mod_test.models import Test, TestPlatform
 
 
+class BlockedUsers(Base):
+    __tablename__ = 'BlockedUsers'
+    # userID refers to the ID from https://api.github.com/users/your_username
+    userID = Column(Integer, primary_key=True)
+
+    def __repr__(self):
+        return "<BlockedUsers(userID='{}')>".format(
```

Comment:
That can be done on a single line. Also, I'd prefer an named argument for the formatting, it'll be a bit more clear.